composer.json

```diff
@@ -24,7 +24,8 @@
     },
     "require-dev": {
         "cakephp/cakephp-codesniffer": "^3.0",
-        "phpunit/phpunit": "~5.7 | ~6.0"
+        "phpunit/phpunit": "~5.7 | ~6.0",
+        "wyrihaximus/twig-view": "4.x-dev as 4.1"
```

Comment:
@WyriHaximus Can you please tag a new release for 4.x so I can replace this with "proper" constraint.